public/app/plugins/datasource/cloud-monitoring/components/MetricQueryEditor.tsx

```diff
@@ -2,7 +2,8 @@ import { useCallback, useEffect } from 'react';
 import * as React from 'react';
 
 import { SelectableValue, TimeRange } from '@grafana/data';
-import { EditorRows, Stack } from '@grafana/experimental';
+import { EditorRows } from '@grafana/plugin-ui';
+import { Stack } from '@grafana/ui';
```

Comment:
Is the `Stack` from `@grafana/ui` the same as the `Stack` from `@grafana/experimental`?

Reply:
that is a good question, i didn't notice and visual difference at a glance, but i will double check.

Reply:
based on the deprecation message for the stack component in grafana/experimental i am entitled to believe these are the same.

```
/**
 * @deprecated use the Stack component from @grafana/ui instead. Available starting from @grafana/ui@10.2.3
 */
```